From cfb92378bca207f338f06961bb0646807db52bb2 Mon Sep 17 00:00:00 2001
From: Marian Chereji <marian.chereji@nxp.com>
Date: Thu, 3 Mar 2016 17:54:11 +0200
Subject: [PATCH 1174/1429] dpa_offload: Add dependency on the CAAM driver

Updated the DPAA offloading driver dependencies by adding the CAAM
Job Ring driver backend dependency. dpa_ipsec driver is using part
of this driver so it counts on it being enabled during the build.

Signed-off-by: Marian Chereji <marian.chereji@nxp.com>
[Original patch taken from QorIQ-SDK-V2.0-20160527-yocto]
Signed-off-by: Yanjiang Jin <yanjiang.jin@windriver.com>
---
 drivers/staging/fsl_dpa_offload/Kconfig |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/drivers/staging/fsl_dpa_offload/Kconfig b/drivers/staging/fsl_dpa_offload/Kconfig
index 708e370..719ff9e 100644
--- a/drivers/staging/fsl_dpa_offload/Kconfig
+++ b/drivers/staging/fsl_dpa_offload/Kconfig
@@ -1,7 +1,7 @@
 
 config FSL_DPA_OFFLOAD
 	tristate "Freescale Datapath Offloading Driver"
-	depends on FSL_SDK_FMAN
+	depends on FSL_SDK_FMAN && CRYPTO_DEV_FSL_CAAM_JR
 	select CRC8
 	select CRC64_ECMA
 	default n
-- 
1.7.5.4

